nestadia-wgpu/src/main.rs

```diff
@@ -562,6 +593,29 @@ fn main() {
             .expect("No rom passed!")
     };
 
+    // Create the audio device
+    let (_audio_stream, audio_stream_handle) = OutputStream::try_default().unwrap();
+    let audio_sink = Sink::try_new(&audio_stream_handle).unwrap();
+
+    // Create the window
+    let event_loop = EventLoop::new();
+    let window = if cfg!(target_os = "windows") {
+        // On windows, rodio / cpal COM initialization conflicts with winit.
+        // Rodio / cpal has to be initialized before winit, and drag and drop has to be disabled
+        // https://github.com/rust-windowing/winit/issues/1255
+        // https://github.com/rust-windowing/winit/issues/1185
+        WindowBuilder::new()
+            .with_title("Nestadia")
+            .with_drag_and_drop(false)
+            .build(&event_loop)
+            .unwrap()
+    } else {
+        WindowBuilder::new()
+            .with_title("Nestadia")
+            .build(&event_loop)
+            .unwrap()
+    };
```

Comment:
This does not compile on linux. I think this would work instead:
```suggestion
    #[cfg(target_os = "windows")]
    let window = 
        // On windows, rodio / cpal COM initialization conflicts with winit.
        // Rodio / cpal has to be initialized before winit, and drag and drop has to be disabled
        // https://github.com/rust-windowing/winit/issues/1255
        // https://github.com/rust-windowing/winit/issues/1185
        WindowBuilder::new()
            .with_title("Nestadia")
            .with_drag_and_drop(false)
            .build(&event_loop)
            .unwrap();

    #[cfg(not(target_os = "windows"))]
    let window = 
        // On windows, rodio / cpal COM initialization conflicts with winit.
        // Rodio / cpal has to be initialized before winit, and drag and drop has to be disabled
        // https://github.com/rust-windowing/winit/issues/1255
        // https://github.com/rust-windowing/winit/issues/1185
        WindowBuilder::new()
            .with_title("Nestadia")
            .build(&event_loop)
            .unwrap();
```

Reply:
True, I noticed too late that this wouldn't compile as the drag and drop function is windows only, and the cfg macro wouldn't help for that. Is it better inline like this, or should I make a "create_window" function with both `#[cfg]`?